### DISCUSSION TITLE
通过iframe将文档嵌入其他页面，右侧文档目录结构无法显示，如何解决

### DISCUSSION THREAD

**Discussion by wanglilin0628** [web-infra-dev/rspress#935] [Q&A] [answered]
通过rspress搭建一个文档网站，并基于iframe方式嵌入已有的vue3工程内，访问文档正常，但是文档右侧的目录树无法展示。

**Comment by Timeless0911** [comment, marked as answer]
This is the default behavior when you use iframe, you can see the logic in https://github.com/web-infra-dev/rspress/blob/16b37fb343af91785eaa6221613f19172d339845/packages/theme-default/src/logic/useUISwitch.ts#L25-L33

The frontMatter config in pages or the themeConfig for all have a higher priority to control the display of aside component, you can set it on your own.

**Reply by wanglilin0628** [reply]
The problem has been solved, thank you。
